app/src/androidTest/java/nextstep/shoppingcart/ui/shoppingcart/ShoppingCartScreenTest.kt

```diff
@@ -0,0 +1,116 @@
+package nextstep.shoppingcart.ui.shoppingcart
+
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onAllNodesWithContentDescription
+import androidx.compose.ui.test.onFirst
+import androidx.compose.ui.test.onNodeWithText
+import androidx.compose.ui.test.performClick
+import nextstep.shoppingcart.data.repository.DefaultShoppingCartRepository
+import nextstep.shoppingcart.domain.model.Product
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+class ShoppingCartScreenTest {
+    @get:Rule
+    val composeTestRule = createComposeRule()
+
+    private val productA =
+        Product(
+            id = 0L,
+            imageUrl =
+                "https://search.pstatic.net/common/?src=http%3A%2F%2Fblogfiles.naver.net" +
+                    "%2FMjAyNDAyMjNfMjkg%2FMDAxNzA4NjE1NTg1ODg5.ZFPHZ3Q2HzH7GcYA1_Jl0lsIdvAnzUF2h6Qd6bgDLHkg." +
+                    "_7ffkgE45HXRVgX2Bywc3B320_tuatBww5y1hS4xjWQg.JPEG%2FIMG_5278.jpg&type=sc960_832",
+            name = "대전 장인약과",
+            price = 12000,
```

Comment:
nitpicking일 수 있지만! 저는 가격을 작성할 때에는 다음과 같이 작성하여 가독성을 챙기는 편입니다

```suggestion
            price = 12_000,
```